### PR TITLE
Use database

### PR DESCRIPTION
**Pull request by mattolson**
* Update sample app to use a sqlite database and SQLAlchemy
* Review logic and refresh endpoints
* Add uninstall callback

### CODE REVIEW

**Comment by mattolson** [comment]
ping @gabelimon

File: app.py [hunk 1]
```diff
@@ -69,63 +101,99 @@ def client_id():
...
+    else:
+        user.email = email
+        user.store = store
+
+    db.session.add(user)
+    db.session.commit()
+
+    # Log user in and redirect to app home
+    flask.session['userid'] = user.id
+    return flask.redirect(flask.url_for('index'))
```
Comment (gabelimon) [inline app.py]: Perhaps for these redirects you could/should pass along a message?

**Comment by gabelimon** [comment]
Completely outside the scope of this PR
I'd like it if we split this out into multiple files but that probably suggests a bit much about suggested app architecture.
Anyway, great job as per usual :+1:

**Comment by mattolson** [comment]
Yeah, this is meant as a bare-bones example of core functionality, so I think keeping it in a single file is ok. Developers will/should know that they'll need a better organizational structure for larger apps.

**Pull request merged**